pkg/config/config_template.yaml

```diff
@@ -123,6 +123,10 @@ api_key:
 # Make sure your client is sending to the same UDP port
 # dogstatsd_port: 8125
 #
+# The host to bind to receive external metrics (dogstatsd and JMX). For
```

Comment:
let's say `used only by the dogstatsd server for now`? (instead of `dogstatsd and JMX`)

JMXFetch uses the same host, but only because it pushes dogstatsd metrics/events to dogstatsd, it doesn't actually binds to the host to listen to anything. What do you think?